pkgs/games/mindustry/default.nix

```diff
@@ -37,6 +37,8 @@ let
   version = "142";
   buildVersion = makeBuildVersion version;
 
+  isWayland = lib.lists.elem "-DGLEW_EGL=ON" glew.cmakeFlags;
```

Comment:
I'd rather not depend on implementation details of `pkgs.glew`. Maybe it's better to have a `enableWayland` attribute, take both glew and glew-egl, and select both the wrapper arguments and the glew package based on `enableWayland`.